.github/workflows/ci-test.yml

```diff
@@ -0,0 +1,43 @@
+name: CI Experiment
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: "master"
+      - name: Setup Java JDK
+        uses: actions/setup-java@v1
+        with:
```

Comment:
With `actions/setup-java@v3` there's also nice caching option; I have this for jackson:

```
      uses: actions/setup-java@v3
      with:
        distribution: 'temurin'
        java-version: ${{ matrix.java_version }}
        cache: 'maven'
```

(matrix just to test with Java 8, 11 and 17, here we can just put `8` in there)

Reply:
What's caching?